pgmq-extension/sql/pgmq--1.4.4--1.5.0.sql

```diff
@@ -0,0 +1,84 @@
+DROP FUNCTION IF EXISTS pgmq.drop_queue(TEXT, BOOLEAN);
+CREATE OR REPLACE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN)
+RETURNS BOOLEAN AS $$
+DECLARE
+    qtable TEXT := pgmq.format_table_name(queue_name, 'q');
+    fq_qtable TEXT := 'pgmq.' || qtable;
+    atable TEXT := pgmq.format_table_name(queue_name, 'a');
+    fq_atable TEXT := 'pgmq.' || atable;
+BEGIN
+    EXECUTE FORMAT(
```

Comment:
```suggestion
BEGIN
    RAISE WARNING "drop_queue(queue, is_partitioned) is deprecated and will be removed in pgmq 2.0. Use drop_queue(queue) instead"
    
    EXECUTE FORMAT(
```

Reply:
Ah, you can't do this because you're calling this function from the other function...

Reply:
It was my request to call this function from the other, just to reduce code duplication. If we'd rather have the deprecation warning, I'd be ok with going back to duplicating that code.

Reply:
What if we move the bulk of the logic to the non-deprecated function, and make the deprecated function call the non-deprecated one?

Reply:
That wouldn't be possible because the non-deprecated function only has 1 argument, while the deprecated one has 2.

Reply:
Of course it is, you just need to ignore the second one

Reply:
Well that's what I proposed initially (https://github.com/tembo-io/pgmq/pull/319#discussion_r1805130178) but it wasn't accepted

Reply:
dropping the parameter is different than ignoring it, right?

What I was saying in https://github.com/tembo-io/pgmq/pull/319#discussion_r1805150303 is that if we drop the function `pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN DEFAULT FALSE)` then it is a breaking change. Making it such the the `partitioned` has the same behavior whether the value is `true` or `false` I think would be ignoring it.

Reply:
@chuckhend So, ideally the way to go is to move the code to the non deprecated function, and have the deprecated function ignore the `partitioned` argument?

Reply:
@ChuckHend Also I was referring to this specific part of the exchange.![Screenshot_20241022-143425_GitHub.png](https://github.com/user-attachments/assets/4031a56e-ef6a-4d49-a17c-7c1822e1261a)

Reply:
I'll push a commit doing the change in your branch

Reply:
@v0idpwn that works for me, thanks.

Reply:
@Neptune650 lol, I tried to push and it got behind somehow. Trying to fix it, sorry about the noise

Reply:
Done. Hopefully it's alright now :)

Reply:
@v0idpwn Looks pretty good to me, let me know if it can be merged yet.